src/plugins/cordova-plugin-inappbrowser/app-host-handlers.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+module.exports = function () {
+    var Browser = require('./inapp-browser'),
+        instance;
+
+    function execute(fnName) {
+        if (!instance) {
+            return;
+        }
+
+        var fn = instance[fnName],
+            args;
+
+        if (Object.keys(arguments).length > 1) {
+            // remove the first element since it is the "fnName" parameter
+            Array.prototype.shift.apply(arguments);
+            args = arguments;
+        }
+
+        return fn.apply(instance, args);
+    }
+
+    return {
+        'InAppBrowser': {
+            'open': function (success, fail, args) {
+                instance = Browser.create(success, fail, args);
+            },
+            'show': function () {
+                execute('show');
+            },
+            'close': function () {
+                execute('close');
```

Comment:
That is my doubt too. I didn't find anything in the spec. I will play with this plugin on a real device and see what is the behaviour. But I think that could be fine to assign undefined or null on close.